proof/hash_test.go

```diff
@@ -137,44 +134,31 @@ func ExampleHashProve_2() {
-	// 000001a0  51 62 5d e6 2b 56 b2 b5  3a e1 c8 8c f7 29 8a 13  |Qb].+V..:....)..|
-	// 000001b0  75 59 98 ea ce f4 6d d5  d0 62 85 51 8e fe d9 4a  |uY....m..b.Q...J|
-	// 000001c0  02 1f 35 03 33 d3 0e 4e  6b b8 fc f9 c9 92 4d e9  |..5.3..Nk.....M.|
-	// 000001d0  c3 1c 35 ec 19 43 7c 25  1b b4 70 09 30 08 e3 a1  |..5..C|%..p.0...|
-	// 000001e0  e1 42 ed 92 0d 82 63 d3  5a 0e 97 78 e6 74 ce a0  |.B....c.Z..x.t..|
-	// 000001f0  24 34 c1 66 7d af 32 9e  59 22 f2 9a 67 3c ea e5  |$4.f}.2.Y"..g<..|
-	// 00000200  4f 54 6d 3e 07 f1 1e 6d  18 7f 8b 95 e3 c4 b9 33  |OTm>...m.......3|
-	// 00000210  ad 94 69 b5 b4 13 b8 51  2f 24 a7 98 e4 06 f4 b2  |..i....Q/$......|
-	// 00000220  f3 ee e8 73 de 78 d1 ab  ff 11 e3 6e df 3d a8 b5  |...s.x.....n.=..|
-	// 00000230  13 86 b6 a5 86 f9 a6 ef  ca 77 46 df 8d 3b eb fb  |.........wF..;..|
-	// 00000240  00 c8 61 cc fd 7a                                 |..a..z|
+	// 00000000  45 85 60 73 be bc 55 10  0e 40 44 59 99 ce 5c 76  |E.`s..U..@DY..\v|
+	// 00000010  f5 ac 0e 6c e6 00 b0 93  01 41 5b e9 9c 39 fe d0  |...l.....A[..9..|
+	// 00000020  65 71 9c 31 f7 b9 ce 81  57 b3 6b 47 41 54 2b d7  |eq.1....W.kGAT+.|
+	// 00000030  f8 15 b6 a2 bc 2d b3 e0  fe c2 77 09 6d 93 b4 69  |.....-....w.m..i|
+	// 00000040  2f ab 85 4d 65 b1 b6 eb  d8 16 96 5f ae 47 38 1d  |/..Me......_.G8.|
+	// 00000050  a7 69 cf 0e 24 04 ff 0a  ec 54 24 4e 09 c0 ec d5  |.i..$....T$N....|
+	// 00000060  c7 e9 cd 3c 93 2b 52 f7  f6 ba bc 89 03 0e bd 2d  |...<.+R........-|
+	// 00000070  bc be 3a d9 b0 5f cf ba  a8 f7 a7 38 57 e3 67 d0  |..:.._.....8W.g.|
+	// 00000080  ef de 78 87 05 20 9a d2  43 2a ff 77 36 62 6a 1a  |..x.. ..C*.w6bj.|
+	// 00000090  53 07 d4 34 a9 d5 b4 39  7d 0b 99 c8 23 76 2e b9  |S..4...9}...#v..|
+	// 000000a0  48 e3 19 0e 76 69 14 0e  9a 6a ef 6c be 4a df af  |H...vi...j.l.J..|
+	// 000000b0  7c 6b 00 8c 7d a0 e4 33  b2 91 cc b4 18 69 ca c0  ||k..}..3.....i..|
+	// 000000c0  fd 83 93 4f ca fa ed 2f  b6 43 27 e3 a5 4b a1 0c  |...O.../.C'..K..|
+	// 000000d0  b3 fb 4b 2c 82 2e 32 a0  83 12 34 f6 c6 8b 93 03  |..K,..2...4.....|
+	// 000000e0  ee f3 b2 f4 06 e4 98 a7  24 2f 51 b8 13 b4 b5 69  |........$/Q....i|
+	// 000000f0  94 ad 33 b9 c4 e3 95 8b  7f 18 6d 1e f1 07 3e 0d  |..3.......m...>.|
+	// 00000100  c3 86 a1 24 1b 3e e1 59  d5 bd 70 a1 ff f9 7c 07  |...$.>.Y..p...|.|
+	// 00000110  8c 9c 52 f7 47 34 46 c9  1a 05 4b 68 57 49 c7 0e  |..R.G4F...KhWI..|
+	// 00000120  31 68 8d ca 3f 6a 85 a1  0d f1 cf 9d 21 05 83 f2  |1h..?j......!...|
+	// 00000130  35 63 b0 65 a8 50 a5 ee  ec 95 f8 fd 78 de 73 08  |5c.e.P......x.s.|
+	// 00000140  35 e7 59 fa 2b 41 20 f8  b6 48 43 62 91 f1 c6 99  |5.Y.+A ..HCb....|
+	// 00000150  0e 64 9c 2c 06 fe 84 75  4f ca 03 7f 28 b5 6d 0c  |.d.,...uO...(.m.|
+	// 00000160  1d 63 e5 73 26 c4 9f 61  62 5b 5c 34 70 66 d0 e4  |.c.s&..ab[\4pf..|
+	// 00000170  ec ca b8 ee 9a 50 07 6b  0c 75 5a a6 77 b3 20 0f  |.....P.k.uZ.w. .|
```

Comment:
I guess it has to do with the small size of Ed25519 scalars/points/signature. If you want to dig deeper, you can check the size of P256 points / scalars.